recipe/meta.yaml

```
@@ -1,5 +1,5 @@
 {% set name = "zstandard" %}
-{% set version = "0.19.0" %}
+{% set version = "0.21.0" %}
 {% set zstd_version = "1.5.2" %}
```

```suggestion
{% set zstd_version = "1.5.5" %}
```
We probably need this version now, but we need https://github.com/conda-forge/zstd-feedstock/pull/69 first.